kdbg/dbgdriver.cpp

```diff
@@ -97,7 +97,7 @@ CmdQueueItem* DebuggerDriver::executeCmdString(DbgCommand cmd,
 	if (m_state == DSrunningLow) {
 	    // take the liberty to interrupt the running command
 	    m_state = DSinterrupted;
-	    ::kill(pid(), SIGINT);
+	    ::kill(processId(), SIGINT);
```

Comment:
Why is this needed?

Reply:
Because of [this](https://doc.qt.io/qt-5/qprocess-obsolete.html#pid) warning.

Reply:
Please do not fix what is not necessary to fix. Removing deprecation warnings is a different topic. Keep your change focused.

Reply:
Sure, will do.